website/conferences/message.py

```diff
@@ -164,3 +160,11 @@ def attachments(self):
                 range(count),
             ),
         )
+
+    @property
+    def find_allowed_types(self):
+        from .model import Conference
+        all_confs = list(Conference.find())
+        allowed_types = [conf.field_names['talk'] for conf in all_confs] + [conf.field_names['poster'] for conf in all_confs]
```

Comment:
You also don't need to loop twice over `all_confs`. Just check for both things in one pass. Simplistic example:

```
allowed_types = []
for conf in all_confs:
    allowed_types.extend( (conf.field_names['thing1'], conf.field_names['thing2'] )
```